src/StatusProvider/ValidateNestedTreeProvider.php

```diff
@@ -8,29 +8,33 @@
 
 use Doctrine\Persistence\ManagerRegistry;
 use Zicht\Bundle\MenuBundle\Entity\MenuItem;
+use Zicht\Bundle\StatusBundle\StatusProvider\StatusProviderHelper;
 use Zicht\Bundle\StatusBundle\StatusProvider\StatusProviderInterface;
 
-/**
- * Class ValidateNestedTreeProvider
- *
- * @package Zicht\Bundle\MenuBundle\StatusProvider
- */
-class ValidateNestedTreeProvider implements StatusProviderInterface
+class ValidateNestedTreeProvider extends StatusProviderHelper implements StatusProviderInterface
```

Comment:
`StatusProviderHelper` is geintroduceerd in `zicht/status-bundle:^1.4.0`

Willen we ergens een check dat deze helper ook echt bestaat?  Of willen we daar vanuit gaan, is al best een tijdje geleden die 1.4.0...

Reply:
Technisch noodzakelijk, maar in de praktijk gaat dit nooit een probleem zijn. Ik zou dat niet hier gaan afdwingen.